features/fields.feature

```diff
@@ -82,97 +74,69 @@ Functionaliteit: Aanpasbare representatie met de fields parameter
     En wordt attribuut naam.voornamen teruggegeven
     En wordt attribuut naam.geslachtsnaam teruggegeven
     En wordt attribuut naam.voorvoegsel teruggegeven
-    En wordt geen enkel ander attribuut dan burgerservicenummer en naam teruggegeven
-    En wordt geen enkele relatie van de resource in _links teruggegeven
     En wordt attribuut _links.self teruggegeven
+    En wordt geen enkel ander attribuut dan burgerservicenummer, naam en _links teruggegeven
+    En bevat _links geen ander attribuut dan self
     En wordt er geen gerelateerde sub-resource teruggegeven in _embedded
 
   Scenario: Een of enkele attributen binnen een groep worden gevraagd
-    Als een ingeschreven persoon wordt geraadpleegd met fields=naam.aanschrijfwijze
+    Als een ingeschreven persoon wordt geraadpleegd met fields=naam.aanschrijfwijze,naam.voornamen
     Dan wordt attribuut naam.aanschrijfwijze teruggegeven
-    En wordt in naam geen enkel ander attribuut dan aanschrijfwijze teruggegeven
-    En wordt geen enkel ander attribuut dan naam en _links teruggegeven
-    En wordt geen enkele relatie van de resource in _links teruggegeven
-    En wordt attribuut _links.self teruggegeven
-    En wordt er geen gerelateerde sub-resource teruggegeven in _embedded
-
-    Als een ingeschreven persoon wordt geraadpleegd met fields=naam.voorvoegsel,naam.geslachtsnaam
-    Dan wordt attribuut naam.geslachtsnaam teruggegeven
-    En wordt attribuut naam.voorvoegsel teruggegeven
-    En wordt in naam geen enkel ander attribuut dan voorvoegsel en geslachtsnaam teruggegeven
-    En wordt geen enkele relatie van de resource in _links teruggegeven
+    En wordt attribuut naam.voornamen teruggegeven
     En wordt attribuut _links.self teruggegeven
+    En bevat naam geen ander attribuut dan aanschrijfwijze en voornamen
+    En bevat _links geen ander attribuut dan self
+    En wordt geen enkel ander attribuut dan naam en _links teruggegeven
     En wordt er geen gerelateerde sub-resource teruggegeven in _embedded
 
   Scenario: Relaties (links) vragen (en beperken) in het antwoord
     Gegeven de te raadplegen persoon heeft een actuele partner(partnerschap of huwelijk), ouders en kinderen
-    En de te raadplegen persoon heeft een BAG-adres (nummeraanduiding) als verblijfplaats
-
-    Als een ingeschreven persoon wordt geraadpleegd met fields=burgerservicenummer,naam,partners
+    Als een ingeschreven persoon wordt geraadpleegd met fields=burgerservicenummer,naam,_links.partners
     Dan wordt attribuut _links.partners teruggegeven
     En wordt attribuut burgerservicenummer teruggegeven
     En wordt attribuut naam.voornamen teruggegeven
     En wordt attribuut naam.geslachtsnaam teruggegeven
     En wordt attribuut naam.voorvoegsel teruggegeven
     En wordt attribuut _links.self teruggegeven
     En wordt in _links geen enkel ander attribuut dan self en partners teruggegeven
```

Comment:
Als we ervoor kiezen om op de ter review aangeboden manier willen beschrijven, dan verwacht ik hier:

En wordt attribuut _links.partners teruggegeven
En wordt geen enkel ander attribuut dan burgerservicenummer, naam en _links teruggegeven

om het consistent te houden met bovenstaande scenario stappen